packages/frontend/src/components/wallet/NFTBox.js

```diff
@@ -130,12 +130,22 @@ const NFTBox = ({ tokenDetails }) => {
             {
                 ownedTokensMetadata &&
                 <div className='tokens'>
-                    {ownedTokensMetadata.map(({ token_id, metadata: { mediaUrl, title } }) => {
+                    {ownedTokensMetadata.map(({ token_id, metadata: { mediaUrl, title } }, index) => {
+                        const videoProps = index === 0 ? { autoPlay: true } : {};
                         return <div className='nft' key={token_id}>
-                            <img src={mediaUrl} alt='NFT' onError={(e) => {
-                                e.target.onerror = null;
-                                e.target.src = FailedToLoad;
-                            }}/>
+                            {
+                                mediaUrl.match(/\.webm$/i)
+                                    ? <video poster="data:image/gif,AAAA" muted={true} loop controls { ...videoProps } autoPlay={true}>
```

Comment:
🔪 this appears to always set the autoplay prop to true; probably left-over from before you added the 'only set autoplay on the tag if it is the first NFT being rendered for the current contract'

Reply:
And yes, you are correct.  Fixed it.